src/TLuaInterpreter.cpp

```diff
@@ -2361,6 +2361,25 @@ int TLuaInterpreter::disableKey( lua_State *L )
     return 1;
 }
 
+int TLuaInterpreter::killKey( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "killKey: wrong argument type" );
+        lua_error( L );
+        return 1;
+    }
+    else
+    {
+        luaSendText = lua_tostring( L, 1 );
```

Comment:
In the medium term, we will replace these `std::string` capturing lines with `QString` ones and replacing the _cargo-cult_ "luaSendText" with more meaningful local names.  In this case I'd suggest using a `QString`:

    itemName = QString::fromUtf8( lua_tostring( L, 1 ) );